src/SourceBuild/tarball/content/src/source-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+    "defaults": {
+        "defaultRef": "main",
+        "exclude": [
+            "**/*.dll",
+            "**/*.Dll",
+            "**/*.exe",
+            "**/*.pdb",
+            "**/*.mdb",
+            "**/*.zip",
+            "**/*.nupkg"
+        ]
+    },
+    "mappings": [
+        {
+            "name": "arcade",
+            "defaultRemote": "https://github.com/dotnet/arcade",
+            "include": [
+                "*.*",
+                ".*",
+                "eng/**/*",
+                "src/**/*",
+                "scripts/**/*"
+            ],
+            "exclude": [
+                "**/tests/**/*",
+                "**/*.Tests/**/*"
+            ]
+        },
+        {
+            "name": "aspnetcore",
+            "defaultRemote": "https://github.com/dotnet/aspnetcore",
+            "exclude": [
+                "**/samples/**/jquery-validation-unobtrusive/.bower.json",
+                "**/samples/**/jquery-validation-unobtrusive/*.js"
+            ]
+        },
+        {
+            "name": "clicommandlineparser",
+            "defaultRemote": "https://github.com/dotnet/clicommandlineparser"
+        },
+        {
+            "name": "command-line-api",
+            "defaultRemote": "https://github.com/dotnet/command-line-api"
+        },
+        {
+            "name": "deployment-tools",
+            "defaultRemote": "https://github.com/dotnet/deployment-tools"
+        },
+        {
+            "name": "diagnostics",
+            "defaultRemote": "https://github.com/dotnet/diagnostics"
+        },
+        {
+            "name": "format",
+            "defaultRemote": "https://github.com/dotnet/format"
+        },
+        {
+            "name": "fsharp",
+            "defaultRemote": "https://github.com/dotnet/fsharp"
+        },
+        {
+            "name": "installer",
+            "defaultRemote": "https://github.com/dotnet/installer",
+            "defaultRef": "vmr"
+        },
+        {
+            "name": "linker",
+            "defaultRemote": "https://github.com/dotnet/linker"
+        },
+        {
+            "name": "msbuild",
+            "defaultRemote": "https://github.com/dotnet/msbuild"
+        },
+        {
+            "name": "nuget-client",
+            "defaultRemote": "https://github.com/NuGet/NuGet.Client",
+            "defaultRef": "dev",
+            "exclude": [
```

Comment:
The underlying issue was resolved (https://github.com/NuGet/NuGet.Client/pull/4740) so these are no longer necessary.